doc-snippets/config-starter-web.md

```diff
@@ -14,7 +14,6 @@
 | `oidc.client.issuer.uri` _string_                         | URI that can either be an OpenID Connect discovery endpoint or an OAuth 2.0 Authorization Server Metadata endpoint defined by RFC 8414.                                         |                                                                                                                                                          | `"https://keycloak.sdadev.sda-se.io/auth/realms/exampleRealm"` | `OIDC_CLIENT_ISSUER_URI`                         |
 | `oidc.client.authentication-passthrough.enabled` _string_ | Flag indicating whether the token from the existing request context should be used, if exists, before attempting to generate new one                                            | `true`                                                                                                                                                   | `false`                                                        | `OIDC_CLIENT_AUTHENTICATION_PASSTHROUGH_ENABLED` |
 | `cors.allowed-origin-patterns` _string_                   | Comma separated list of URL patterns for which CORS requests are allowed.                                                                                                       | _none allowed_                                                                                                                                           | `"https://*.all-subdomains.com, https://static-domain.com"`    | `CORS_ALLOWEDORIGINPATTERNS`                     |
-| `request.body.max.size` _size_                            | The maximum size allowed for request body data sent by a client.                                                                                                                | _1 MB_                                                                                                                                                   | `100 KB`, `10MB`                                               | `REQUEST_BODY_MAX_SIZE`                          |
```

Comment:
I think this should be mentioned in the release notes. Can we add this to the commit message? Will the semantic release plugin add commit message details?